R/data_read.R

```diff
@@ -158,6 +158,23 @@ data_read <- function(path,
 # process imported data from SPSS, SAS or Stata -----------------------
 
 .post_process_imported_data <- function(x, convert_factors, verbose) {
+  # for "unknown" data formats (like .RDS), which still can be imported via
```

Comment:
shouldn't this be directly  in `.read_unknown()` since it's the only case where the import can be other than a dataframe? 

If we import an SAS or SPSS file and don't have a dataframe, we should error and not warn